products/google-cloud-sql-proxy.md

```diff
@@ -0,0 +1,37 @@
+---
+# Name of the product (mandatory).
+title: Cloud SQL Auth Proxy
+category: service
+tags: database google server-app service
+iconSlug: googlecloud
+permalink: /cloud-sql-auth-proxy
+alternate_urls:
+-   /cloud-sql-proxy
+versionCommand: cloud-sql-proxy --version
+releasePolicyLink: https://github.com/GoogleCloudPlatform/cloud-sql-proxy?tab=readme-ov-file#support-policy
+changelogTemplate: "https://github.com/GoogleCloudPlatform/cloud-sql-proxy/releases/tag/__LATEST__"
+eolColumn: Support
+
+# Auto-update release configuration (optional).
+# This is used for automatically updating `releaseDate`, `latest`, and `latestReleaseDate` for every release.
+# Multiple configurations are allowed.
+# Please visit https://github.com/endoflife-date/endoflife.date/wiki/Automation for more details.
+# The presence of such configuration modifies the product page so that users are informed that existing
+# releases are automatically updated with latest versions.
+auto:
+  methods:
+    - git: https://github.com/GoogleCloudPlatform/cloud-sql-proxy.git
+      regex: ^v(?P<major>\d+)_(?P<minor>\d+)_(?P<patch>\d{1,3})_?(?P<tiny>\d+)?$
+
+releases:
+-   releaseCycle: "2"
+    releaseDate: 2023-01-17
```

Comment:
this part has to be 2025-04-28 according to here: https://github.com/GoogleCloudPlatform/cloud-sql-proxy/releases/tag/v2.16.0